client/src/templates/Challenges/video/Show.tsx

```diff
@@ -322,6 +315,16 @@ export const query = graphql`
   query VideoChallenge($slug: String!) {
     challengeNode(fields: { slug: { eq: $slug } }) {
       videoId
+      videoLocaleIds {
+        espanol
+        italian
+        portuguese
+      }
```

Comment:
To confirm, as we roll out new languages we'll also need to add them here?

Reply:
Right. Anything we want a localised video for we'd need to add to this query.

Reply:
Made a PR to add this to the docs.